be/src/common/config.h

```diff
@@ -1087,6 +1087,28 @@ CONF_Bool(datacache_tiered_cache_enable, "true");
 CONF_String(datacache_engine, "");
 // The interval time (millisecond) for agent report datacache metrics to FE.
 CONF_mInt32(report_datacache_metrics_interval_ms, "60000");
+// Whether enable automatically adjust cache space quota.
+// If true, the cache will choose an appropriate quota based on the current remaining space as the quota.
+// and the quota also will be changed dynamiclly.
+// Once the disk space usage reach the urgent level, the quota will be decreased to keep the disk usage
+// around the disk safe level.
+// On the other hand, if the cache is full and the disk usage falls below the disk safe level for a long time,
+// which is configured by `datacache_disk_free_period`, the cache quota will be increased to keep the disk usage
+// around the disk safe level.
+CONF_mBool(datacache_auto_adjust_enable, "false");
```

Comment:
datacache_enable_auto_adjust better? Verb before noun

Reply:
It's just a suggestion. If it's not easy to change, forget it.

Reply:
> datacache_enable_auto_adjust better? Verb before noun

Historical issues, we need to keep consistence with old names.